keps/sig-api-machinery/3158-optional-maximum-limit-on-list-requests-to-etcd/kep.yaml

```diff
@@ -0,0 +1,54 @@
+title: Optional maximum limit on list requests to etcd
+kep-number: 3158
+authors:
+  - "@chaochn47"
+owning-sig: sig-api-machinery
+participating-sigs:
+  - sig-scalability
+status: provisional
+creation-date: 2022-04-05
+reviewers:
+  - "@wojtek-t"
+  - "@shyamjvs"
+  - "@anguslees"
+  - "@ptabor"
+  - "@serathius"
+approvers:
+  - "@deads2k"
+  - "@lavalamp"
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: implementable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.25"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.25"
+  beta: tbd
+  stable: tbd
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: MaximumListLimitOnEtcd
```

Comment:
A clearer name would be `MaximumPageSizeLimitForEtcdLists` imo. Let's see if others have better suggestions.

Reply:
That looks good to me.